src/inverseRobot.js

```diff
@@ -25,6 +25,16 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (invertedRobot[robot[key]]) {
```

Comment:
i think better to do variables for if ' robot[key] ', and use .hasOwnProperty for checking :)